### PR TITLE
Remove use of AAD Graph for cloud shell

### LINKED ISSUE #849: Migrate from AAD graph to MS graph
AAD graph (graph.windows.net) will be retired on or after 30 September without notice. Need to migrate to MS graph (graph.microsoft.com).

### PR DESCRIPTION
**Pull request by alexweininger**
Closes #849 

AAD Graph was being used by cloud shell in two ways:

1. If a user had subscriptions from multiple tenants, we let the user select which tenant to launch the cloud shell in. A Graph API endpoint for listing user tenants was used to populate the quick pick.

    I replaced this with a call to the new List Tenants ARM endpoint. https://learn.microsoft.com/en-us/rest/api/resources/tenants/list

2. We fetched an AAD Graph token and passed it to Cloud Shell upon initialization.

    I just stopped fetching and passing this token. My reasoning is that any features relying on this token will also break once AAD Graph is gone. So there's no reason to keep passing this token.

### CODE REVIEW

**Review by bwateratmsft — APPROVED** [review]

**Comment by alexweininger** [comment]
Waiting on some confirmation from the cloud shell team on these changes before merging.

**Pull request merged**
